packages/backend/src/index.ts

```diff
@@ -23,8 +23,8 @@ app.use(
     credentials: true
   })
 )
-app.use(express.json())
-app.use(express.urlencoded({ extended: true }))
```

Comment:
when backside of ID is also being sent to rapyd, we get PayloadTooLargeError, so this is to fix that error